models/articles-models.js

```diff
@@ -17,12 +11,11 @@ const fetchID = (id) => {
 };
 const fetchArticles = () => {
   let queryStr =
-    "SELECT articles.*, COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id";
+    "SELECT articles.*, COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id  ORDER BY created_at desc";
```

Comment:
Nice job, remember you may have to change this for a later task where you accept different order_by and sort_by

Reply:
How can i get this last pull back to my VS code i've somehow lost all of the new work